packages/modal/package.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
This is tricky because it _is_ the styles that supported `sp-dialog-wrapper` previously, which is why there's not readme and no component, as it's all used therein still. Our Spectrum CSS processing allows us to make multiple styles from a single pattern, but currently only allows one pattern per package, so I couldn't think of a better way to support this without loosing functionality. I intend to use `sp-modal` to address some of the questions around overlay functionality in separate future work, but knew that definitely didn't belong here.